### PR TITLE
Majed Resume

### PR DESCRIPTION
**Pull request by saidsag**

### CODE REVIEW

File: index.htm [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Majed-CV"</title>
```
Comment (rabeedameer) [inline index.htm]: "fix the Title Please"

File: index.htm [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      <h2>Education</h2>
+      <p>12 Years Studying in the hiegh School</p>
+      <p> <time datetime="2017-04">04-2017</time>German language course B2 level </p>
+
+    </section>
+
+    <section>
+
+
+
```
Comment (rabeedameer) [inline index.htm]: "You do not need all this space between elements"

File: index.htm [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      <h2>Languages</h2>
+      <ul>
+        <li>Arabic</li>
+        <li>English</li>
+        <li>German</li>
+        <li>Turkish</li>
+
+
```
Comment (rabeedameer) [inline index.htm]: "the same as the former comment regarding space"

File: index.htm [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      </ul>
+    </section>
+    <section>
+      <h2>IT Skills</h2>
+      <p>linux, git, HTML5, CSS</p>
```
Comment (rabeedameer) [inline index.htm]: It would be better if you use Abbreviations for CSS and HTML

File: index.htm [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    <ul>
+      <li>Video Games</li>
+      <li>Travelling</li>
+      <li>Football</li>
+      <li>Languages learning</li>
+    </ul>
+  </main>
+
+  <footer>
+    <p>&copy;coded by Said</p>
```
Comment (rabeedameer) [inline index.htm]: It is nice to use it and would be better if it contains other info about Majed ;)

File: index.htm [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+  </footer>
+</body>
+
+</html>
```
Comment (rabeedameer) [inline index.htm]: you do not need a new line after </html>

File: index.htm [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+</head>
+
+<body>
+  <header>
+    <h1>Majed Shrendan</h1>
+    <h2>Informations</h2>
+    <ul>
+      <li>birthday <time datetime="1995-01-04">04/01/1995</time></li>
+      <li>Telefon: 00963 940217311</li>
+      <li> <a href="mailto:majedshrndan45@gmail.com?subject=feedback">email me</a> </li>
```
Comment (rabeedameer) [inline index.htm]: "try to fix this link please cuz you need to show the email not a link to it"

**Review by rabeedameer — CHANGES_REQUESTED** [review]
"nice job but you need to take care of some fixes"

**Comment by saidsag** [comment]
can y check it please?

File: index.htm [hunk 8]
```diff
@@ -5,52 +5,50 @@
...
+
   <header>
     <h1>Majed Shrendan</h1>
     <h2>Informations</h2>
     <ul>
       <li>birthday <time datetime="1995-01-04">04/01/1995</time></li>
       <li>Telefon: 00963 940217311</li>
-      <li> <a href="mailto:majedshrndan45@gmail.com?subject=feedback">email me</a> </li>
-      <li>Address: Pfeffer weg 6, 13589 Berlin</li>
+      <li> <a href="mailto:majedshrndan45@gmail.com">email me</a> </li>
```
Comment (rabeedameer) [inline index.htm]: change "email me "

File: index.htm [hunk 9]
```diff
@@ -59,10 +57,8 @@ <h2>Hobbies</h2>
       <li>Languages learning</li>
     </ul>
   </main>
-
   <footer>
-    <p>&copy;coded by Said</p>
-
+    <p>&copy;coded by Said for Majed Shrndan</p>
```
Comment (rabeedameer) [inline index.htm]: sorry, you still need to change!

**Review by rabeedameer — COMMENTED** [review]
some fixes

**Review by rabeedameer — APPROVED** [review]
Thanks for your efforts!

**Pull request merged**